src/pybind/mgr/stats/fs/perf_stats.py

```diff
@@ -14,7 +14,7 @@
 from threading import Lock, Condition, Thread, Timer
 from ipaddress import ip_address
 
-PERF_STATS_VERSION = 1
+PERF_STATS_VERSION = 2
```

Comment:
requires changing version checking in cephfs-top script.

Reply:
Done!!